superglue/lib/utils/react.ts

```diff
@@ -1,15 +1,11 @@
-import {
-  visit,
-  remote,
-  saveAndProcessPage,
-  copyPage,
-} from '../action_creators'
+import { visit, remote, saveAndProcessPage, copyPage } from '../action_creators'
+import { Content, PageOwnProps, RootState } from '../types'
 import { urlToPageKey } from './url'
 
 export function mapStateToProps(
-  state = { pages: {}, superglue: {} },
-  ownProps
-) {
+  state: RootState,
```

Comment:
I removed the default states here as `mapStateToProps` would never receive  and empty state that looks like the defaults 
build output:
```diff
 function mapStateToProps(state, ownProps) {
-  if (state === void 0) {
-    state = {
-      pages: {},
-      superglue: {}
-    };
-  }
```